src/renderer/components/notification-comp.tsx

```diff
@@ -1,10 +1,11 @@
-import classNames from 'classnames';
 import { ipcRenderer } from 'electron';
 import * as React from 'react';
 
 import { i18n } from '../../common/i18n-preload';
 
 const whiteColorRegExp = new RegExp(/^(?:white|#fff(?:fff)?|rgba?\(\s*255\s*,\s*255\s*,\s*255\s*(?:,\s*1\s*)?\))$/i);
+const darkTheme = [ '#b5616a', '#ab8ead', '#ebc875', '#a3be77', '#58c6ff', '#ebab58' ];
+type Theme = '' | 'light' | 'dark';
```

Comment:
do we really need an empty string here?

Reply:
Yes, this is because of the `1.5` dependency